packages/lit-element/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
```

Comment:
We should set this up as a composite project with a reference to lit-html. We'll get automated rebuilds of lit-html when necessary :)